service-statistics-unsd/src/main/java/fi/nls/oskari/control/statistics/plugins/unsd/UnsdConfig.java

```diff
@@ -0,0 +1,58 @@
+package fi.nls.oskari.control.statistics.plugins.unsd;
+
+import fi.nls.oskari.log.LogFactory;
+import fi.nls.oskari.log.Logger;
+import org.json.JSONObject;
+
+public class UnsdConfig {
+
+    private final static Logger LOG = LogFactory.getLogger(UnsdConfig.class);
+
+    private long datasourceId;
+    private String url;
+    private String goal = "1";
+    private String timeVariableId = "timePeriod";
+
+    public UnsdConfig() {
+        this(new JSONObject(), -1l);
+    }
+
+    public UnsdConfig(JSONObject json, long id) {
+        datasourceId = id;
+        init(json);
+    }
+
+    private void init(JSONObject json) {
+        url = json.optString("url", null);
+        // allow override with db config
+        goal = json.optString("goal", goal);
+        timeVariableId = json.optString("timeVariable", timeVariableId);
+    }
+
+    public long getId() {
+        return datasourceId;
+    }
+    public String getUrl() {
+        if(url == null) {
+            // default
```

Comment:
Is there a reason for `url` field having a different logic for default value compared to `goal` and `timeVariableId` fields that I can not spot?

Reply:
Not really no -> will change this to work the same way